sdks/python/apache_beam/transforms/window_test.py

```diff
@@ -149,7 +146,7 @@ def test_sessions(self):
     result = (pcoll
               | 'w' >> WindowInto(Sessions(10))
               | GroupByKey()
-              | sort_values
+              | Map(lambda (k, vs): (k, sorted(vs)))
```

Comment:
It turns out that Pytest introduces a global view in test collecting phase. Using `lambda` globally causes it unpickled. Since global variable `Map(lambda (k, vs): (k, sorted(vs)))` is only used once in this module, there is no necessary to create a variable separately.